presto-elasticsearch/pom.xml

```diff
@@ -386,6 +386,7 @@
                 <configuration>
                     <ignoredDependencies>
                         <ignoredDependency>org.yaml:snakeyaml:jar</ignoredDependency>
+                        <ignoredDependency>org.elasticsearch:elasticsearch-x-content:jar</ignoredDependency>
```

Comment:
```
[ERROR] Non-test scoped test only dependencies found:
[ERROR]    org.elasticsearch:elasticsearch-x-content:jar:7.17.27:compile
```
On making it test scoped.
```
[ERROR] Found a problem with test-scoped dependency org.elasticsearch:elasticsearch-x-content
  Scope compile was expected by artifact: org.elasticsearch:elasticsearch:7.17.27

```

Reply:
This is a recent change. I have updated the `jetty-upgrade` branch with this fix